app/views/groups/show.html.erb

```diff
@@ -5,7 +5,7 @@
 		<p class="fw-bold">$<%= @group.total_amount.round(2) %></p>
   </div>
   <div class="p-4 pt-0">
-    <%=image_tag @group.icon, class: 'img-thumbnail ratio ratio-21x9' %>
+    
     <p class="text-muted text-end pt-2"><%= @group.name %></p>
   </div>
 </div>
```

Comment:
- [ ] Please kindly ensure that the login and signup buttons on the splash screen matches the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/198050130-812f2883-a57c-437f-be4d-a80207c0cdfe.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/198050525-462812e5-0625-4b1b-acea-fc40f46b7e7b.png)